apps/modernization-ui/src/validation/entry/checkEmptyRule.ts

```diff
@@ -0,0 +1,6 @@
+export const checkEmptyRule = {
+    pattern: {
+        value: /.*[^ ].*/gm,
```

Comment:
Question: I don't think this does what we want. Double check because `.*` means 0 or more of any character.

So think what we have here is `0 or more of any character - not a space - 0 or more of any character`

Reply:
I could be wrong but I think we need it to match anything other a space, so that the validation passes. The type of character check is done with another validation, so this regex would focus on not matching with a input of only spaces.